subprojects/build-init/src/main/resources/org/gradle/buildinit/tasks/templates/library-versions.properties

```diff
@@ -6,7 +6,7 @@ testng=6.10
 slf4j=1.7.22
 guava=21.0
 scala=2.11
-groovy=2.4.8
+groovy=2.4.9
```

Comment:
Done and that's really good to know. Can the comment be edited to reflect that the file shouldn't be changed?